WcaOnRails/app/mailers/competitions_mailer.rb

```diff
@@ -64,20 +64,20 @@ def notify_organizer_of_removal_from_competition(remover, competition, organizer
   def notify_users_of_results_presence(user, competition)
     @competition = competition
     @user = user
-    mail(
-      to: user.email,
-      subject: "The results of #{competition.name} are posted",
-      reply_to: competition.all_delegates.pluck(:email),
-    )
+
+    localized_mail user.locale || :en,
```

Comment:
@viroulep also, a question on this. If `user.locale` is not `:en` and it doesn't have the translations yet, would this default to the `en` translation or should I leave this as `:en` until we have the translations?

Reply:
We have a fallback to `en` whenever the key doesn't exist in the requested locale, so this line is perfectly fine!
We also have a check that the set of keys used in the website matches exactly the set of keys defined by the English locale, so that we never have missing/unused keys.